lib/pdf2img.js

```diff
@@ -16,11 +16,11 @@ var options = {
 var Pdf2Img = function() {};
 
 Pdf2Img.prototype.setOptions = function(opts) {
-  options.type = opts.type || options.type;
```

Comment:
This doesn't seem right.

Reply:
Thanks for the feedback, I made another commit that should handle assignments on the options variable.